### PR TITLE
Amp 558

### PR DESCRIPTION
**Pull request by vinita84**

### CODE REVIEW

File: tools/video_tools/run-tesseract.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+from pytesseract import Output
+
+def main():
+	(input_file, output_name) = sys.argv[1:3]
+	os.mkdir(input_file[:-4])
+	dateTimeObj = datetime.now()
+
+	#ffmpeg part starts here
+	command = "ffmpeg -i "+input_file+ " -an -vf fps=2 '"+input_file[:-4]+"/frame_%05d_"+str(dateTimeObj)+".jpg'"
+	subprocess.call(command, shell=True)
```
Comment (yingfeng-iu) [inline tools/video_tools/run-tesseract.py]: shall we add some error handling if ffmpeg fails for some reason?

File: tools/video_tools/run-tesseract.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+from datetime import datetime
+from decimal import Decimal
+from pytesseract import Output
+
+def main():
+	(input_file, output_name) = sys.argv[1:3]
+	os.mkdir(input_file[:-4])
+	dateTimeObj = datetime.now()
+
+	#ffmpeg part starts here
```
Comment (yingfeng-iu) [inline tools/video_tools/run-tesseract.py]: could you add a comment about what the ffmpeg is doing here (something like "generate jpg files per frame")?

File: tools/video_tools/run-tesseract.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      		#save frame if it had text
+		if len(frameList["boundingBoxes"]) > 0:
+			output["frames"].append(frameList)
+  
+	with open(output_name, 'w') as outfile:
+		json.dump(output, outfile)
+	pp = pprint.PrettyPrinter(indent=2)
+	pp.pprint(output)
+
+
```
Comment (yingfeng-iu) [inline tools/video_tools/run-tesseract.py]: the MGM team doesn't require any json conversion to standard format, right?

File: tools/video_tools/run-tesseract.py (hunk 2 above)
Comment (vinita84) [reply tools/video_tools/run-tesseract.py]: done

**Pull request merged**